submissions/yhorodechna/html-css-popup/index.html

```diff
@@ -0,0 +1,137 @@
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/picture.png" alt="" />
+                                        <span class="subtitle">Picture</span>
+                                    </a>
+                                </li>
+                                <li class="popup__item">
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/plane.png" alt="" />
+                                        <span class="subtitle">Plane</span>
+                                    </a>
+                                </li>
+                                <li class="popup__item">
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/shopping-cart.png" alt="" />
+                                        <span class="subtitle">Shopping</span>
+                                    </a>
+                                </li>
+                                <li class="popup__item">
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/write.png" alt="" />
+                                        <span class="subtitle">Note</span>
+                                    </a>
+                                </li>
+                                <li class="popup__item">
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/responsive.png" alt="" />
+                                        <span class="subtitle">Responsive</span>
+                                    </a>
+                                </li>
+                                <li class="popup__item">
+                                    <a href="#" class="popup__link">
+                                        <img class="img" src="img/statistics.png" alt="" />
+                                        <span class="subtitle">Statistics</span>
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <nav class="menu">
+                <ul class="menu__list">
+                    <li><a class="menu__item" href="#"><img class="img" src="img/header/search.png" alt="" /></a>
+                    </li>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/notification.png" alt="" /></a>
+                    </li>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/user.png" alt="" /></li></a>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/settings.png" alt="" /></a>
+                    </li>
```

Comment:
For the future don't forget about consistency: if you use `#` as a value for `href` attribute you should use it for all empty `href` attributes)

Reply:
Ok, thanks)